test/module/irohad/multi_sig_transactions/mst_processor_test.cpp

```diff
@@ -343,7 +345,7 @@ TEST_F(MstProcessorTest, receivedOutdatedState) {
                                            std::make_shared<TestCompleter>());
   const auto expired_batch = makeTestBatch(txBuilder(1, time_before, 3));
   transported_state += addSignaturesFromKeyPairs(expired_batch, 0, makeKey());
-  mst_processor->onNewState(another_peer_key, transported_state);
+  mst_processor->onNewState(another_peer_key, std::move(transported_state));
```

Comment:
let's restrict the scope of moved-from variable
```suggestion
  {
    auto transported_state = ...
    mst_processor->onNewState(another_peer_key, std::move(transported_state));
  }
```